recipes/cv_explainability/requirements.txt

```diff
@@ -1,6 +1,6 @@
 openvino-dev==2023.2
 nncf==2.7.0
```

Comment:
openvino and nncf should be updated to 2024.0 and 2.9.0. Otherwise, I have exceptions while exporting.